.github/workflows/tests.yml

```diff
@@ -26,31 +26,44 @@ jobs:
     - uses: actions/checkout@v2
       with:
         fetch-depth: 0
+
     - name: Set up Python ${{ matrix.python }}
       uses: actions/setup-python@v2
       with:
         python-version: ${{ matrix.python }}
+
     - name: Install
       run: |
         python -m pip install --upgrade pip
         python -m pip install -e ".[tests,scripts]" --use-deprecated=legacy-resolver
+
     - name: Print environment
       run: |
         python -m pip freeze
         python --version
         python -c "import pysparkling; print(pysparkling.__version__)"
+
+    - name: Check if import order is fine
+      run: |
+        isort . --check
```

Comment:
When I test this locally, it doesn't actually work:

```
WARNING: Unable to parse file . due to [Errno 21] Is a directory
```

Reply:
That is because you're not using the latest version of isort. I was using this too, and was wondering why it did not work. An upgrade made it work. I think this change is introduced in 2.6.0. You should `pip install .[tests,scripts,sql]`. I believe I also adjusted in this PR the versions needed.